src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -137,7 +153,14 @@ void initializeReference() {
      */
     void initializeReads() {
         if (! readArguments.getReadFiles().isEmpty()) {
-            SamReaderFactory factory = SamReaderFactory.makeDefault().validationStringency(readArguments.getReadValidationStringency());
+            // Prefetcher is useful for cloud files because of the latencies involved.
+            boolean gsPathPresent = readArguments.getReadPaths().stream()
+                .filter(p->p.toUri().getScheme().equals("gs")).findAny().isPresent();
```

Comment:
While we're at it we may want to handle the non-index case, though that requires a change to htsjdk (samReaderFactory.open(samPath, wrapper))